CMakeLists.txt

```diff
@@ -4,4 +4,6 @@ project(Burgers_Static VERSION 1.0 LANGUAGES C Fortran)
 
 find_package(OpenCMISSLibs 1.3.0 REQUIRED COMPONENTS Iron CONFIG)
 
+include(OCMiscFunctions)
```

Comment:
Why did you add this in?  I don't see any of the functions defined in this file being used.